nodes/exchange/FCStd_read.py

```diff
@@ -0,0 +1,102 @@
+from sverchok.dependencies import FreeCAD
+from sverchok.utils.dummy_nodes import add_dummy
+
+if FreeCAD is None:
+    add_dummy('SvReadFCStdNode', 'SvReadFCStdNode', 'FreeCAD')
+
+else:
+    F = FreeCAD
+    import bpy,sys
+    from bpy.props import IntProperty, FloatProperty, StringProperty, BoolProperty
+    from sverchok.node_tree import SverchCustomTreeNode
+    from sverchok.utils.logging import info, exception
+    from numpy import ndarray
+
+    class SvReadFCStdNode(bpy.types.Node, SverchCustomTreeNode):
+        ''' SvReadFCStdNode '''

```

Comment:
We have a new convention for the nodes docstrigns:

https://github.com/nortikin/sverchok/blob/a95871e3a5a77b13e63fda4654a07a638f6ab19d/nodes/field/attractor_field_mk2.py#L28-L31
The triggers appear on the search menu. The tooltip when you place the mouse over the node name in the menu